packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -99,17 +69,29 @@ class HttpRequestData {
     return data;
   }
 
-  final int _timelineMicrosBase;
+  static const _connectionInfoKey = 'connectionInfo';
+  static const _contentTypeKey = 'content-type';
+  static const _cookieKey = 'cookie';
+  static const _errorKey = 'error';
+  static const _filterKey = 'filterKey';
```

Comment:
nit: normalize the key names. This one has the word `Key` in the value but others don't.

Reply:
filterKey is actually the name of the key in the arguments map. This would need to be changed on the VM but I think it is actually an accurate description as is because it is describing a key to group events on.